### DISCUSSION TITLE
Icons in release package (RPM) - tabby-1.0.213-linux-x64

### DISCUSSION THREAD

**Discussion by VieleBerge** [Eugeny/tabby#9950] [General]
The icons included with tabby in tabby-1.0.213-linux-x64.rpm under /usr/share/icons/hicolor/apps/ could use some work, there is a 512x512 folder, but that contains tabby.png in size 1024x1024, the icon in 32x32 seems to be correct, I would love to contribute and maybe add the missing icon sizes, just point me in the right direction, and tell me if there is anything I need to know about this, before making a PR

**Comment by Eugeny** [comment]
I'm using `electron-builder` for packaging, which is using the different icon sizes that are pregenerated in `build/icons` - feel free to see if anything can be improved there :+1:

**Comment by VieleBerge** [comment]
It seems to be fixed, at least when building locally, the .rpm have all common icon sizes included, and the icon sizes matches the folder they are located in. How do you generate the releases? Either its something in that process, or something got fixed on main between generating the release RPM and me building locally 🏔️

**Reply by Eugeny** [reply]
The release is generated using the `scripts/build-macos.js` script - you can check out the Github workflow definitions in the `.github` folder
